dj/main/migrations/0009_auto_20170819_2302.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-19 23:02
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0008_auto_20161014_1424'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='episode',
+            name='reviewers',
+            field=models.TextField(blank=True, help_text='email(s) of the reviewers(s)'),
+        ),
+        migrations.AlterField(
+            model_name='episode',
+            name='edit_key',
+            field=models.CharField(blank=True, default='58088430', help_text='key to allow unauthenticated users to edit this item.', max_length=32, null=True),
```

Comment:
default='58088430',    
in model.py: default = str(random.randint(10000000,99999999)),

It should be somewhat hard to guess.   It lets people who know it bump the state into "make public" without a login.